sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -170,6 +170,16 @@ object SQLConf {
     }
   }
 
+  /**
+   * The map contains info about removed SQL configs. Keys are SQL config names,
+   * map values are tuples of default config value and the version in which
+   * the SQL config was removed.
+   */
+  val removedSQLConfigs = Map[String, (String, String)](
```

Comment:
Oh, yeah. I was thinking about having this mechanism in SQL configurations too but couldn't find some time to fix.
There is an example to follow in `SparkConf`, for example, see `deprecatedConfigs`.

Can you match how it looks like? Then it should be good to go.

Reply:
Just to add more info, `deprecatedConfigs` works like `removedSLQConfigs` (because they are deprecated first and removed out).